backend/src/openarchiefbeheer/destruction/api/permissions.py

```diff
@@ -20,6 +20,30 @@ def has_permission(self, request, view):
             "accounts.can_review_destruction"
         ) or request.user.has_perm("accounts.can_review_final_list")
 
+    def has_object_permission(self, request, view, destruction_list):
```

Comment:
There is a frontend counterpart of this in `permissions.ts`, can you check if its in sync with this implementation?

In an ideal world we would be able to validate that automatically.